ci/define_pvs.yml

```
@@ -0,0 +1,49 @@
+---
+- name: "Define Pvs"
+  hosts: controller
+  vars:
+   base_dir: "{{ ansible_user_dir }}"
+  tasks:
+    - name: "Log into the cluster"
+      ansible.builtin.import_role:
+        name: rhol_crc
+        tasks_from: add_crc_creds.yml
+
+    - name: Check if the install_yamls already exists
+      ansible.builtin.stat:
+        path: "{{ base_dir }}/install_yamls"
+      register: check_install_yamls_dir
+
+    - name: "Get Install yaml repo"
+      when: not check_install_yamls_dir.stat.exists
+      ansible.builtin.git:
+        repo: https://github.com/openstack-k8s-operators/install_yamls
+        dest: "{{ base_dir }}/install_yamls"
+   
+    - name: Check if make command exists
+      ansible.builtin.command: "command -v make"
+      register: make_check_result
+      ignore_errors: true
+
+    - name: Install make if not present
+      ansible.builtin.package:
+        name: make
+        state: present
+      when: make_check_result.rc != 0
```

The package module should only install make if it doesn't already exist.
So this check and when are not needed

```suggestion
    - name: Install make if not present
      ansible.builtin.package:
        name: make
        state: present
```

---

I forgot that, thanks emma 😅